src/spotted/config/yaml/settings.yaml.types

```diff
@@ -18,5 +18,6 @@ post:
   delete_anonymous_comments: bool
   reject_after_autoreply: bool
   autoreplies_per_page: int
+  blacklist_message: list[str]
```

Comment:
By the way, this could be changed by adding more supported types in

https://github.com/UNICT-DMI/Telegram-SpottedDMI-Bot/blob/8ee62f309c1f8d841dcba8b19eed16ce13b74e24/src/spotted/data/config.py#L250

since right now, it only supports a few basic types and lists

https://github.com/UNICT-DMI/Telegram-SpottedDMI-Bot/blob/8ee62f309c1f8d841dcba8b19eed16ce13b74e24/src/spotted/data/config.py#L265

`list[str]` and `list[int]` could make for great additions.

Reply:
for now I will leave as it is